### PR TITLE
docs: Add OIDC prompt attribute

### PR DESCRIPTION
**Pull request by Dan-Heath**
Per ICU-11466, we are adding the OIDC `prompts` attribute to support authentication prompts including account picker functionality. This PR updates the domain model and CLI documentation with the new options (https://hashicorp.atlassian.net/browse/ICU-11903).

View the updates in the preview deployment:

- [OIDC domain model auth method attributes](https://boundary-h766xzy27-hashicorp.vercel.app/boundary/docs/concepts/domain-model/auth-methods#oidc-auth-method-attributes)
- CLI
   - [auth method create](https://boundary-h766xzy27-hashicorp.vercel.app/boundary/docs/commands/auth-methods/create#usages-by-type) on the OIDC tab
   - [auth method update](https://boundary-h766xzy27-hashicorp.vercel.app/boundary/docs/commands/auth-methods/update#usages-by-type) on the OIDC tab

### CODE REVIEW

File: website/content/docs/commands/auth-methods/update.mdx [hunk 1]
```diff
@@ -251,6 +251,21 @@ The following are options are specific to OIDC auth-methods in addition to the c
 
 - `-max-age` `(string: "")` - Indicates the OIDC "max_age" parameter sent to the provider.
 
+- `prompts` `(string: "")` - Indicates whether the OIDC authorization server should display authentication or consent user interface prompts.
+You can optionally configure one or more of the following types of prompts to customize the behavior of the authentication process:
+   - `none` - The authorization server does not display any authentication or consent prompts.
+   - `login` - The authorization server prompts users for reauthentication before allowing them to log in.
+   - `consent` - The authorization server prompts users for consent before returning any information to Boundary.
+   - `select_account` - The authorization server prompts users to select a user account.
+   The `select_account` option can be helpful if your users have multiple sets of login credentials.
```
Comment (elimt) [inline website/content/docs/commands/auth-methods/update.mdx:260]: Should we say `multiple accounts` here rather?

File: website/content/docs/commands/auth-methods/update.mdx [hunk 2]
```diff
@@ -251,6 +251,21 @@ The following are options are specific to OIDC auth-methods in addition to the c
 
 - `-max-age` `(string: "")` - Indicates the OIDC "max_age" parameter sent to the provider.
 
+- `prompts` `(string: "")` - Indicates whether the OIDC authorization server should display authentication or consent user interface prompts.
```
Comment (elimt) [inline website/content/docs/commands/auth-methods/update.mdx:254]: Should we say: "display reauthentication, account selection or consent user prompts"? Just to highlight the `select_account` prompt

File: website/content/docs/commands/auth-methods/update.mdx (hunk 1 above)
Comment (Dan-Heath) [reply website/content/docs/commands/auth-methods/update.mdx]: Yes, that's much cleaner, thanks! I changed it on all 3 pages.

**Review by elimt — APPROVED** [review]
LGTM

**Review by stellarsquall — APPROVED** [review]
Nice work!

**Pull request merged**